### PR TITLE
test: add bytea type

### PR DESCRIPTION
**Pull request by svysali**
Reproduces the user-api password hash failure. Test fails at the moment

### CODE REVIEW

**Review by bernielomax — APPROVED** [review]
Lgtm.

File: axon_sql.go [hunk 1]
```diff
@@ -40,16 +40,7 @@ func prepareQueryArgs(changesetCols []*ChangesetColumn) ([]string, []string, map
...
-		// in the database, so that we can use it to lookup column types by column
-		// name. That will allow us to handle things like `bytea` specially.
-		//
-		// Swap the commented line below for the uncommented one to see the tests pass
-
-		// `getColumnType(columnName)` is a hypothetical helper fuction that can lookup
-		// a columns type name by column name.
-		//if c.Value != nil && getColumnType(c.Column) == "bytea" {
-		if c.Value != nil && c.Column == "type_bytea" {
+		if c.Value != nil && c.Type == "bytea" {
```
Comment (13rac1) [inline axon_sql.go:43]: 🚀

File: axon_sql.go [hunk 2]
```diff
@@ -99,7 +99,7 @@ func prepareUpdateQuery(primaryKey []string, change *Changeset) (string, map[str
 	}
 	setClauses := make([]string, len(cols))
 	for i, c := range cols {
-		setClauses[i] = fmt.Sprintf("%s = :%s", c, c)
+		setClauses[i] = fmt.Sprintf("%s = %s", c, colArgs[i])
```
Comment (13rac1) [inline axon_sql.go:93]: 👀

File: axon_schema.go [hunk 3]
```diff
@@ -219,3 +222,70 @@ func updateOrphanSequences(sourceDB *sqlx.DB, targetDB *sqlx.DB, table string, c
...
+	return nil
+}
+
+// getColumnType fetches the  data type for a column in the database
+func getColumnType(schema, table, column string) (string, error) {
+
+	tableName := fmt.Sprintf(`"%s"."%s"`, schema, table)
+	colType, ok := columnTypes[tableName][column]
+	if !ok {
+		return "", fmt.Errorf(`column type not found for column %s in table "%s"."%s"`, schema, table, column)
```
Comment (bernielomax) [inline axon_schema.go]: The arg order looks incorrect.

**Review by bernielomax — APPROVED** [review]
This looks pretty good to me. Just the one comment.

File: axon_schema.go (hunk 3 above)
Comment (svysali) [reply axon_schema.go]: fixed

**Pull request merged**